package/MDAnalysis/core/selection.py

```diff
@@ -512,16 +512,8 @@ def __init__(self, parser, tokens):
     def apply(self, group):
         mask = np.zeros(len(group), dtype=np.bool)
         for val in self.values:
-            if val.count('*') > 1:
-                raise SelectionError('Can only use one wildcard in a string')
-            wc_pos = val.find('*')
-            if wc_pos == -1:  # No wildcard found
-                mask |= getattr(group, self.field) == val
-            else:
-                values = getattr(group, self.field).astype(np.str_)
-                mask |= np.char.startswith(values, val[:wc_pos])
-                mask &= np.char.endswith(values, val[wc_pos+1:])
-
+            values = getattr(group, self.field).astype(np.str_)
```

Comment:
so I think I was calling `.astype(np.str_)` here because we were pushing it into a `np.char` function.  Now we're using `fnmatch` this likely isn't necessary, try removing this?